src/adagrams.js

```diff
@@ -1,7 +1,109 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const letterQuantity = {"A": 9, "B": 2, "C": 2, "D": 4, "E": 12, "F": 2, "G": 3, "H": 2, "I": 9, "J": 1, "K": 1, "L": 4, "M": 2, "N": 6, "O": 8, "P": 2, "Q": 1, "R": 6, "S": 4, "T": 2, "U": 4, "V": 2, "W": 2, "X": 1, "Y": 2, "Z": 1};
+
+    let letterPool = [];
+
+    // method for doing something 'x' times
+    const times = x => f => {
+      if (x > 0) {
+        f();
+        times (x - 1) (f);
```

Comment:
Minor note: remove space between function and arguments to increase readability.

```suggestion
        times(x - 1) (f);
```

Reply:
Fun use of recursion!